src/pages/dev/reference/glossary.mdx

```diff
@@ -0,0 +1,59 @@
+# Axelar Glossary
```

Comment:
I'm still a little confused why we are adding a glossary that only has ITS terms separately from the ITS docs?

Reply:
This is the start of the Axelar glossary. We will continue adding more terms and using them as a comprehensive reference.